api/samples/CMakeLists.txt

```diff
@@ -312,13 +313,14 @@ DR_install(TARGETS ${tgts} DESTINATION ${INSTALL_SAMPLES_BIN})
 DR_install(FILES ${srcs} DESTINATION ${INSTALL_SAMPLES})
 DR_install(FILES "${public_clist}"
   DESTINATION "${INSTALL_SAMPLES}" RENAME CMakeLists.txt)
-if (WIN32)                                                    # NON-PUBLIC
-  # have a copy of dynamorio library for tracedump            # NON-PUBLIC
-  get_target_property(DR_TARGET_LOCATION dynamorio            # NON-PUBLIC
-    LOCATION${location_suffix})                               # NON-PUBLIC
-  DR_install(FILES "${DR_TARGET_LOCATION}"                    # NON-PUBLIC
-    DESTINATION "${INSTALL_SAMPLES_BIN}")                     # NON-PUBLIC
-endif (WIN32)                                                 # NON-PUBLIC
+if (WIN32)                                                         # NON-PUBLIC
+  # have a copy of dynamorio library for tracedump                 # NON-PUBLIC
```

Comment:
style: Please add cap + end punct to clearly separate from next line

Reply:
Done